database/seeders/BoardSeeder.php

```diff
@@ -610,26 +669,29 @@ public function run()
-                ['id' => $people + 12, 'Parent' => 'Words', 'color' => $default_color],
-                ['id' => $people + 13, 'Mom' => 'Speak', 'color' => $default_color],
-                ['id' => $people + 14, 'Dad' => 'Say', 'color' => $default_color],
-                ['id' => $people + 15, 'Grandma' => 'Word', 'color' => $default_color],
-                ['id' => $people + 16, 'Grandpa' => 'Words', 'color' => $default_color],
-                ['id' => $people + 17, 'Granparent' => 'Speak', 'color' => $default_color],
-                ['id' => $people + 18, 'Aunt' => 'Say', 'color' => $default_color],
-                ['id' => $people + 19, 'Uncle' => 'Speak', 'color' => $default_color],
-                ['id' => $people + 20, 'Cousin' => 'Say', 'color' => $default_color],
-                ['id' => $people + 21, 'Brother' => 'Say', 'color' => $default_color],
-                ['id' => $people + 22, 'Sister' => 'Speak', 'color' => $default_color],
-                ['id' => $people + 23, 'Sibling' => 'Say', 'color' => $default_color],
-                ['id' => $people + 24, 'Niece' => 'Say', 'color' => $default_color],
-                ['id' => $people + 25, 'Nephew' => 'Speak', 'color' => $default_color],
-                ['id' => $people + 26, 'Nibling' => 'Say', 'color' => $default_color],
-                ['id' => $people + 27, 'Daughter' => 'Say', 'color' => $default_color],
-                ['id' => $people + 28, 'Son' => 'Speak', 'color' => $default_color],
-                ['id' => $people + 29, 'Child' => 'Say', 'color' => $default_color]
+                ['id' => $people + 11, 'text' => 'Family', 'color' => $default_color],
+                ['id' => $people + 12, 'text' => 'Parent', 'color' => $default_color],
+                ['id' => $people + 13, 'text' => 'Mom', 'color' => $default_color],
+                ['id' => $people + 14, 'text' => 'Dad', 'color' => $default_color],
+                ['id' => $people + 15, 'text' => 'Grandma', 'color' => $default_color],
+                ['id' => $people + 16, 'text' => 'Grandpa', 'color' => $default_color],
+                ['id' => $people + 17, 'text' => 'Granparent', 'color' => $default_color],
+                ['id' => $people + 18, 'text' => 'Aunt', 'color' => $default_color],
+                ['id' => $people + 19, 'text' => 'Uncle', 'color' => $default_color],
+                ['id' => $people + 20, 'text' => 'Cousin', 'color' => $default_color],
+                ['id' => $people + 21, 'text' => 'Brother', 'color' => $default_color],
+                ['id' => $people + 22, 'text' => 'Sister', 'color' => $default_color],
+                ['id' => $people + 23, 'text' => 'Sibling', 'color' => $default_color],
+                ['id' => $people + 24, 'text' => 'Niece', 'color' => $default_color],
+                ['id' => $people + 25, 'text' => 'Nephew', 'color' => $default_color],
+                ['id' => $people + 26, 'text' => 'Nibling', 'color' => $default_color],
```

Comment:
```suggestion
 ['id' => $people + 26, 'text' => 'Sibling', 'color' => $default_color],
```

Reply:
I think Nibling was intended.
According to Merriam Webster, "Nibling is a gender-neutral term used to refer to a child of one's sibling as a replacement for 'niece' or 'nephew'."

Reply:
elvis is correct

Reply:
Sibling is next to brother and sister, nibling was intended per the definition elvis gave